content/lua/concepts/tables/terms/create/create.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'create()'
+description: 'Returns a new table with the provided value repeated the provided amount of times.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development' 
+Tags:
+  - 'Tables'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+ - 'paths/computer-science'
+---
+
+The **`.create()`** function generates a new table with a specified value repeated a given number of times.
+
+## Syntax
+
+```lua
+table.create(value, amount)
+```
+
+- `value` : This parameter represents the value to populate the table with.
+- `amount` : This parameter specifies how many times to repeat the `value` in the table.
+
```

Comment:
Let's add a note here explaining what happens when `value` is nill or is not given

```suggestion

> **Note:** When `value` is omitted or `nil`, the table will be returned empty but will have preallocated space for the `amount` specified. Preallocation is done on the array portion of the table, making `table.create` counter-productive on dictionaries.

```